cmd/cap-release/promotioncheck.go

```
@@ -0,0 +1,55 @@
+		for _, problem := range problems {
+			fmt.Fprintf(os.Stderr, "  - %s\n", problem.Error())
+		}
+		return 1
+	}
+	fmt.Printf("promotion-check OK: %s resolves to %s with bound release metadata\n", m.Release.Tag, commit)
+	return 0
+}
+
+func resolveLocalTagCommit(repoRoot, tag string) (string, error) {
+	ref := "refs/tags/" + tag + "^{commit}"
+	cmd := exec.Command("git", "-C", repoRoot, "rev-parse", "--verify", ref)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)